### ISSUE TITLE
[READ] Before reporting about your issue...

### ISSUE THREAD

**Issue by wf9a5m75** [mapsplugin/cordova-plugin-googlemaps#1479]
Hi, thank you for using the cordova-googlemaps-plugin.

If you come here, you may have some issues.
I would like to help you to solve your issues and improve this plugin as much as possible.
But in order to that, I need also your help as much as possible.

## take a screen capture or video record

Please take a screen capture or take a video to describe your issue.
I need to know what is wrong.
For example, crash your app, please take a video when your app crash.

## save the logs

If you are familiar with Android Studio or Xcode, please save the error logs, and provide it.
The 80% of issues, I can understand what happens when I see the logs.
Please provide all logs, not a few lines.

## provide your app code 

The most important thing of debugging is **reproduce your issue**.
In order to that, please share your **whole project files**.
This is the fastest and valuable resource for debugging.

I know you don't want to publish your source code.
So that's why please share your code using [Bitbucket](https://bitbucket.org/). 

Bitbucket provides private git repository service as free.
You can share your project files with only me.
If you are not familiar with git repository, please watch this video.
https://www.youtube.com/watch?v=BtEvnE79jxY

## Need NDA? I can do it.

Sometimes, your company might prohibit that except contracting NDA.
If your company require NDA, ask me. I can agree it.

Basically, I'm not interested in your app or source code. So I don't share your source code with another person.

## Don't send your code as zip file or text file.

Please provide your project files.

-------------

Thank you for your help.

**Comment by jlcarvalho** [comment]
Hey man, you can create templates for issues and pull requests. This can guide users in a more concisely way than this issue.

See: https://github.com/blog/2111-issue-and-pull-request-templates

**Comment by wf9a5m75** [comment]
Thank you for letting me know. I will read it!

**Comment by wf9a5m75** [comment]
This page has been moved to https://github.com/mapsplugin/cordova-plugin-googlemaps-doc/blob/master/articles/before_post_your_issue/README.md
